src/client/components/pages/Courses/MeetingModal.tsx

```diff
@@ -314,6 +314,22 @@ const MeetingModal: FunctionComponent<MeetingModalProps> = function ({
     }
   };
 
+  /**
+   * A handler to delete a meeting from the current existing meetings of the
+   * course instance. If the deleted meeting was being edited at the time of
+   * deletion, the state of the currently edited meeting is set back to null so
+   * that no meetings are expanded in edit mode.
+   */
+  const removeMeeting = (meeting: CourseInstanceResponseMeeting) => {
+    if (currentEditMeeting !== null) {
+      setCurrentEditMeeting(null);
```

Comment:
This also needs to call `setShowRoomsData(null);` to clear the room selection table. Otherwise, if you have a meeting being edit and you've clicked "Show Rooms" to bring up the table before deleting it, the table will persist.

Reply:
Oh great point! I did not think of this scenario. I cleared the room selection table in commit 
8a8d427.